libr/arch/p/lua/lua53_parser.c

```diff
@@ -5,14 +5,18 @@
 // XXX global
 // R_UNUSED static RList *lua53_function_list = NULL;
 
-struct {
+typedef struct lua_data_struct {
+	ut8 ver;
+	bool islittleendian;
```

Comment:
shorter to read/type and easier to read imho. also the other fields are camelcase, so we can follow the same convention here

```suggestion
	bool isLE;
```